scrapers/peeonher.yml

```diff
@@ -0,0 +1,28 @@
+name: peeonher
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - peeonher.com/updates/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+     Title:
+        selector: //h1[@class='page_title']
+     Date:
+        selector: //strong[text()="Published"]/following-sibling::text()
+        postProcess:
+          - replace:
+            - regex: (\d+)[dhnrst]+
+              with: $1
+          - parseDate: 2 January 2006
+     Performers:
+      Name: //div[@class="pull-left"]//a/text()
+     Details: //div[@class='con_left']/p[1]
+     Tags:
+        Name: //a[@class="sub_sub_tag"]//a/text()
```

Comment:
although it somehow seems to work is it a typo or something (the //a[]//a part) ?
```yaml
      Tags:
        Name: //a[@class="sub_sub_tag"]
```
should be ok

Reply:
No idea how that was even in there to be honest.